src/infra/marketplace/marketplaceCmdRepo.go

```diff
@@ -437,6 +428,155 @@ func (repo *MarketplaceCmdRepo) InstallItem(
 	)
 }
 
+func (repo *MarketplaceCmdRepo) generateFindWithMoveCmd(
+	srcDirPath valueObject.UnixFilePath,
+	destinationDirPath valueObject.UnixFilePath,
+	filesToFilter []valueObject.UnixFileName,
+	shouldIgnoreFiles bool,
+) string {
+	params := ""
+	if len(filesToFilter) > 0 {
+		fileNameFilterParams := "-name \"" + filesToFilter[0].String() + "\""
+		for _, fileToIgnore := range filesToFilter[1:] {
+			fileNameFilterParams += " -o -name \"" + fileToIgnore.String() + "\""
+		}
+		params = "\\( " + fileNameFilterParams + " \\)"
+	}
+
+	flags := []string{"-maxdepth 1"}
+	if shouldIgnoreFiles {
+		flags = append(flags, "-mindepth 1", "-not")
+	}
+	flagsStr := strings.Join(flags, " ")
+
+	return fmt.Sprintf(
+		"find %s/ %s %s -exec mv -t %s {} +",
+		srcDirPath.String(),
+		flagsStr,
+		params,
+		destinationDirPath.String(),
+	)
+}
+
+func (repo *MarketplaceCmdRepo) uninstallSymlinkFilesRemoval(
+	installedItem entity.MarketplaceInstalledItem,
+	catalogItem entity.MarketplaceCatalogItem,
+	removalDestinationPath valueObject.UnixFilePath,
+) error {
+	publicDirBackupPath, err := valueObject.NewUnixFilePath("/app/html-backup")
```

Comment:
This naming is probably due to the previous version of the code. You're recreating the "installedItem.InstallDirectory.String()" path and not the public path as you were before. Therefore, this directory as well as the error message on line 510, 515 and 530 are outdated. I'd recommend reading the entire method to make sure it makes sense. You're casting some variables to .String() multiple times without using a variable with suffix Str for example.

Reply:
Done!